engine/access/rpc/backend/backend_events.go

```diff
@@ -50,6 +50,11 @@ func (b *backendEvents) GetEventsForHeightRange(
 		endHeight = head.Height
 	}
 
+	// limit min height to last sealed block in the chain
+	if head.Height < startHeight {
```

Comment:
a fix for issue - https://github.com/onflow/flow-js-sdk/issues/491#issuecomment-770001583

Reply:
I wonder if it would be better to return an error instead. If a user submits a query starting at block height 10, they probably **don't** want to receive any events before that height. This might return them events from block 8. 

It might even make sense to keep it how it was before (not return an error) -- I think there's an argument that an empty result is valid behaviour when `startHeight > sealedHeight`.